### PR TITLE
player-mpv: shows current music track or movie title with underline as progress bar

### PR DESCRIPTION
**Pull request by vyachkonovalov**

### CODE REVIEW

File: polybar-scripts/player-mpv/mpv-control.sh [hunk 1]
```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env bash
```
Comment (x70b1) [inline polybar-scripts/player-mpv/mpv-control.sh]: Can we include the function of this script into the python script as command line parameters? Then we only have one script file.

Comment (vyachkonovalov) [reply polybar-scripts/player-mpv/mpv-control.sh]: Technically we can, but that leads to code complexity (btw, player-mpv.py code already too branchy). Also, the reason mpv-control.sh is a separate bash script is to load it as quick as posible. For operations like time-pos (seek forward/backword), which by default happends on mouse scroll, the speed is important.

Comment (x70b1) [reply polybar-scripts/player-mpv/mpv-control.sh]: `player-mpris-tail` is 4 times longer, I don't think it's too complex.

Is the speed really noticable?

Comment (vyachkonovalov) [reply polybar-scripts/player-mpv/mpv-control.sh]: Will check soon.

Comment (x70b1) [reply polybar-scripts/player-mpv/mpv-control.sh]: Would be perfect! :+1:

Comment (vyachkonovalov) [reply polybar-scripts/player-mpv/mpv-control.sh]: The speed isn't noticable. I didn't know python is fast😃

Comment (x70b1) [reply polybar-scripts/player-mpv/mpv-control.sh]: Speed is relative. We work with scripts here. They are fast and easy to customize. If we would need execution speed, we would do things in C.  :+1:

File: polybar-scripts/player-mpv/readme.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
+# Script: player-mpv
```
Comment (x70b1) [inline polybar-scripts/player-mpv/readme.md]: Lets rename the script and the directory to `player-mpv-tail`.

File: polybar-scripts/player-mpv/readme.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+## Configuration
+
+### 1. *~/.config/mpv/mpv.conf*
+```
+input-ipc-server=/tmp/mpvsocket
+```
+or run mpv with `--input-ipc-server=/tmp/mpvsocket` option.
+
+### 2. Use narrow font like 'Fira Sans Compressed'
```
Comment (x70b1) [inline polybar-scripts/player-mpv/readme.md]: Remove the font suggestion.

**Comment by x70b1** [comment]
Thanks @vyachkonovalov!

**Pull request merged**